etl/steps/data/garden/un_sdg/2022-07-07/un_sdg.py

```diff
@@ -0,0 +1,334 @@
+import pandas as pd
+import numpy as np
+import json
+from structlog import get_logger
+from pathlib import Path
+from typing import Tuple, List, Any, Dict
+from owid.catalog.utils import underscore
+
+from owid.walden import Catalog
+from owid.catalog import Dataset, Table
+from etl.paths import DATA_DIR
+
+COUNTRY_MAPPING_PATH = (
+    Path(__file__).parent / "un_sdg.country_mapping.json"
+).as_posix()
+
+BASE_URL = "https://unstats.un.org/sdgapi"
+VERSION = Path(__file__).parent.stem
+FNAME = Path(__file__).stem
+NAMESPACE = Path(__file__).parent.parent.stem
+
+COUNTRY_MAPPING_PATH = (
```

Comment:
If you want to be consistent with constants above then this could also be `Path(__file__).parent / "un_sdg.country_mapping.json"`